specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-preview/examples/redeemSubscription.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "subscriptionId": "subscriptionId",
+    "api-version": "2021-01-01-preview",
+    "body": {
+      "properties": {
+        "managementGroupId": "testManagementGroupId",
+        "tags": {
+          "tag1": "testTag"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Subscription/subscriptions/subscriptionId",
+        "name": "subscriptionId",
+        "type": "Microsoft.Subscription/subscriptions",
+        "properties": {
+          "subscriptionId": "subscriptions/291bba3f-e0a5-47bc-a099-3bdcb2a50a05"
```

Comment:
subscriptionId should be either guid value or /subscriptions/291bba3f-e0a5-47bc-a099-3bdcb2a50a05